src/util/metrics/threads_linux.rs

```diff
@@ -143,11 +166,37 @@ impl Collector for ThreadsCollector {
                         write_total.inc_by(write_delta);
                     }
                 }
+
```

Comment:
Could we create an independent function to collect the context switch metrics?

Reply:
> Could we create an independent function to collect the context switch metrics?

I think we should create a function for each metric if we do this......

Reply:
got it